app/scaffold-config/tabConfig.js

```diff
@@ -0,0 +1,56 @@
+define(['config/baseConfig'], function(BaseConfig) {
```

Comment:
They are different now:
`menuConfig` is the multi-level navigation structure used to define navitron
`tabConfig` is the old `menuConfig`